### DISCUSSION TITLE
Leave your feedback 💜

### DISCUSSION THREAD

**Discussion by fenske** [DevSkillsHQ/discusssions#5] [Feedback]
Enjoyed your DevSkills experience or have any constructive feedback instead? [We're happy to hear from you](https://github.com/orgs/DevSkillsHQ/discussions/new?category=feedback).

**Comment by alex88g** [comment]
Im glad I participated in your project !

**Reply by fenske** [reply]
Thx for the feedback, @alex88g! 🙏

**Comment by no13bus** [comment]
Hi:
I just want to say that if developer use the ViteJS and cypress in the project, when GitHub action runs, you will came across problems.
The error message is 
```
CypressError: `cy.visit()` failed trying to load:

http://localhost:5173/

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: connect ECONNREFUSED 127.0.0.1:5173
```

Github action not starting Vite server properly.

Here is the solution. The reference:
https://github.com/cypress-io/github-action/issues/879#issuecomment-1519045733

**Comment by alex88g** [comment]
Hi Daniel!

Thank you for your respond. I vill fix the issues as soon as possible.

Have a beautiful day.

Alexander Gallorini

sön 7 maj 2023 kl. 20:44 skrev DanielJia ***@***.***>:
